### DISCUSSION TITLE
no vpn for containers via opnsense

### DISCUSSION THREAD

**Discussion by Kieros** [Jip-Hop/jailmaker#61] [Q&A]
I have opnsense running and created a vpn. 
I have setup aliases to go through with unbound.
It is working for all devices on the LAN except the containers running with jailmaker are not having internet access.
I have set jailmaker up with macvlan for the containers and everything is working fine on the network and pingable.
All containers are listed in the table with their own IP in the same LAN where the other devices are working behind the VPN.
So they are part of the same network. But it seems they are not realy part of it.

Any ideas? I have tried so many things.
I have setup an alias for the WG_hosts (wiregauard using IPs) whenever I add a device from the LAN to it it works, but a container is not.
If I setup open VPN in opnsense with default setup everything is pulled through (forced) to go through the vpn and then somehow it works. 

If you need more info let me know and any help is MUCH appreciated.

**Comment by Kieros** [comment]
As a side note all containers that are not part of the WG_hosts alias are having internet access.

**Comment by Jip-Hop** [comment]
Sorry I am not familiar with the setup you describe. But since you mention unbound maybe it's related to DNS? Which DNS servers are your containers using?

**Comment by Jip-Hop** [comment]
I just confirmed a container made with jailmaker accesses the internet through a Wireguard VPN running on an OpenWrt router without issues. I used bridge networking instead of macvlan.

**Reply by Kieros** [reply]
Ok thank you for you answer
